docs/dictionary/command/go.lcdoc

```diff
@@ -82,6 +89,10 @@ go stack "My Stack"
 
 Otherwise, you must include the stack's file path.
 
+If you don't include a file path the <go> command will look in
+the defaultFolder. If you're in the IDE (not a standalone) the
+defaultFolder will probably start off as a system folder.
```

Comment:
I think this should be rephrased slightly to 
```
 Otherwise, you must specify the stack's file path.

If the specified file path is relative, the <go> command will look in the <defaultFolder>.
```
It is the responsibility of the entry for the `defaultFolder` to describe what it starts off as in various modes (indeed it does do this), so I would remove the last sentence here. You'll need to add `defaultFolder` to the list of references.